.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             features: vendored
             target: x86_64-unknown-linux-gnu
             cache_id: vendored
-          - toolchain: 1.43.0 # MSRV
+          - toolchain: 1.56.0 # MSRV
```

Comment:
Why is this required?

Reply:
The actual version of ```cstr_core``` does not work with the 1.43 version (Alternative would be pinning to a pre 0.2.6 version of ```cstr_core```).

Reply:
MSRV only applies for `std` builds, for `no-std` we anyway require nightly.

Reply:
I think given recent stabilization of `core::ffi` we can just start using that instead on no-std builds.